main/main.cpp

```diff
@@ -1248,7 +1248,7 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 		}
 	}
 
-	GLOBAL_DEF("display/window/force_right_to_left_layout_direction", false);
+	GLOBAL_DEF("internationalization/rendering/force_right_to_left_layout_direction", false);
```

Comment:
I don't know if "rendering" is a good name for this subsection, maybe "gui"? @bruvzg 
Can still be changed in a follow up PR though, not blocking for this one.

Reply:
Maybe move it to "debug/", it intended for testing RTL layouts only.